test/RelayClient.test.ts

```diff
@@ -1578,81 +1592,56 @@ describe('RelayClient', function () {
           fakeMaxPossibleGas
         );
 
-        expect(callStub).to.be.calledOnce;
         await expect(verifyWithRelayHub).to.be.fulfilled;
+        expect(transactionStub).to.be.calledOnce;
       });
 
-      it('should throw if enveloping request will fail due to not enabled worker', async function () {
-        const error = ethersLogger.makeError(
-          'Not an enabled worker',
-          errors.CALL_EXCEPTION
-        );
-        const callStub = sandbox.stub().throws(error);
-        relayHubStub = {
-          callStatic: {
-            relayCall: callStub,
-          },
-        } as unknown as typeof relayHubStub;
-        sandbox.stub(RelayHub__factory, 'connect').returns(relayHubStub);
-        const verifyWithRelayHub = relayClient._verifyWithRelayHub(
-          relayWorkerAddress,
-          FAKE_RELAY_TRANSACTION_REQUEST,
-          fakeMaxPossibleGas
-        );
-
-        expect(callStub).to.be.calledOnce;
-        await expect(verifyWithRelayHub).to.be.rejectedWith(error.message);
-      });
-
-      it('should throw if enveloping request will fail due to invalid gas price', async function () {
```

Comment:
Why did we remove this test?

Reply:
This tests were included before since the method interacted directly with the contract, but now there is an extra layer that does it, I could include it again.

Reply:
updated